scripts/setup_cloud_run_env.sh

```
@@ -0,0 +1,320 @@
+echo "Reading configuration from: $ENV_FILE"
+echo ""
+
+# Load .env file (handle comments and empty lines)
+set -a  # Export all variables
+# More robust .env parsing that handles inline comments
+while IFS= read -r line || [[ -n "$line" ]]; do
+    # Skip empty lines and comments
+    [[ -z "$line" || "$line" =~ ^[[:space:]]*# ]] && continue
+    # Remove inline comments and export
+    line="${line%%#*}"
+    [[ -n "$line" ]] && export "$line"
```

Shellcheck (0.11.0)

[warning] This does not export 'line'. Remove $/${} for that, or use ${var?} to quiet.

(SC2163)